pom.xml

```diff
@@ -66,19 +68,21 @@
       explicitly specifying the latest version here because one we get from the parent POM
       tends to lag behind a bit
     -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
+    <mesos.version>0.21.1</mesos.version>
+    <protobuf.version>2.5.0</protobuf.version>
   </properties>
 
   <dependencies>
       <dependency>
           <groupId>org.apache.mesos</groupId>
           <artifactId>mesos</artifactId>
-          <version>0.20.0</version>
+          <version>${mesos.version}</version>
```

Comment:
As per previous note, if this was used, it would only really be for local trialling of different mesos versions for experimentation. I would never expect it to be built and released this way